package.json

```diff
@@ -1,92 +1,101 @@
 {
```

Comment:
@imlucas sorry for messing with the formatting of the file, actually VS Code did it automatically. We can revert the changes if it creates merge conflicts.

Reply:
No worries! I was just fidgeting :p `.vscode` auto-formatting on save is very intentional